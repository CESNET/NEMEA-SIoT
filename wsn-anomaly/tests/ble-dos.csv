uint64 INCIDENT_DEV_ADDR,time TIME,double aclPackets,double address,double rxAcls,double rxBytes,double rxErrors,double rxEvents,double rxScos,double scoMtu,double scoPackets,double txAcls,double txBytes,double txCmds,double txErrors,double txScos
1112,2018-04-04T10:46:06.000,10.000000,198695763786629.000000,3.000000,2285.000000,156.000000,156.000000,0.000000,64.000000,8.000000,3.000000,6999.000000,143.000000,0.000000,0.000000
1112,2018-04-04T10:46:11.000,10.000000,198695763786629.000000,4.000000,2314.000000,157.000000,157.000000,0.000000,64.000000,8.000000,4.000000,7028.000000,143.000000,0.000000,0.000000
1112,2018-04-04T10:46:16.000,10.000000,198695763786629.000000,5.000000,2343.000000,158.000000,158.000000,0.000000,64.000000,8.000000,5.000000,7057.000000,143.000000,0.000000,0.000000
1112,2018-04-04T10:46:21.000,10.000000,198695763786629.000000,5.000000,2343.000000,158.000000,158.000000,0.000000,64.000000,8.000000,5.000000,7057.000000,143.000000,0.000000,0.000000
1112,2018-04-04T10:46:26.000,10.000000,198695763786629.000000,6.000000,2372.000000,159.000000,159.000000,0.000000,64.000000,8.000000,6.000000,7086.000000,143.000000,0.000000,0.000000
1112,2018-04-04T10:46:36.000,10.000000,198695763786629.000000,7.000000,2401.000000,160.000000,160.000000,0.000000,64.000000,8.000000,7.000000,7115.000000,143.000000,0.000000,0.000000
1112,2018-04-04T10:46:41.000,10.000000,198695763786629.000000,8.000000,2430.000000,161.000000,161.000000,0.000000,64.000000,8.000000,8.000000,7144.000000,143.000000,0.000000,0.000000
1112,2018-04-04T10:46:51.000,10.000000,198695763786629.000000,9.000000,2459.000000,162.000000,162.000000,0.000000,64.000000,8.000000,9.000000,7173.000000,143.000000,0.000000,0.000000
1112,2018-04-04T10:47:01.000,10.000000,198695763786629.000000,10.000000,2488.000000,163.000000,163.000000,0.000000,64.000000,8.000000,10.000000,7202.000000,143.000000,0.000000,0.000000
1112,2018-04-04T10:47:06.000,10.000000,198695763786629.000000,11.000000,2517.000000,164.000000,164.000000,0.000000,64.000000,8.000000,11.000000,7231.000000,143.000000,0.000000,0.000000
1112,2018-04-04T10:47:16.000,10.000000,198695763786629.000000,13.000000,2575.000000,166.000000,166.000000,0.000000,64.000000,8.000000,13.000000,7289.000000,143.000000,0.000000,0.000000
1112,2018-04-04T10:47:21.000,10.000000,198695763786629.000000,15.000000,2633.000000,168.000000,168.000000,0.000000,64.000000,8.000000,15.000000,7347.000000,143.000000,0.000000,0.000000
1112,2018-04-04T10:47:26.000,10.000000,198695763786629.000000,18.000000,2720.000000,171.000000,171.000000,0.000000,64.000000,8.000000,18.000000,7434.000000,143.000000,0.000000,0.000000
1112,2018-04-04T10:47:31.000,10.000000,198695763786629.000000,20.000000,2778.000000,173.000000,173.000000,0.000000,64.000000,8.000000,20.000000,7492.000000,143.000000,0.000000,0.000000
1112,2018-04-04T10:47:36.000,10.000000,198695763786629.000000,22.000000,2836.000000,175.000000,175.000000,0.000000,64.000000,8.000000,22.000000,7550.000000,143.000000,0.000000,0.000000
1112,2018-04-04T10:47:41.000,10.000000,198695763786629.000000,25.000000,2923.000000,178.000000,178.000000,0.000000,64.000000,8.000000,25.000000,7637.000000,143.000000,0.000000,0.000000
1112,2018-04-04T10:47:46.000,10.000000,198695763786629.000000,27.000000,2981.000000,180.000000,180.000000,0.000000,64.000000,8.000000,27.000000,7695.000000,143.000000,0.000000,0.000000
1112,2018-04-04T10:47:51.000,10.000000,198695763786629.000000,29.000000,3039.000000,182.000000,182.000000,0.000000,64.000000,8.000000,29.000000,7753.000000,143.000000,0.000000,0.000000
1112,2018-04-04T10:47:56.000,10.000000,198695763786629.000000,31.000000,3097.000000,184.000000,184.000000,0.000000,64.000000,8.000000,32.000000,7840.000000,143.000000,0.000000,0.000000
1112,2018-04-04T10:48:01.000,10.000000,198695763786629.000000,34.000000,3184.000000,187.000000,187.000000,0.000000,64.000000,8.000000,34.000000,7898.000000,143.000000,0.000000,0.000000
