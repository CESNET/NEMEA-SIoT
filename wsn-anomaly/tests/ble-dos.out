52.200000,1112,23.200000,2018-04-04T10:47:26.000,7,"The device 1112 exceeded the grow up limit with value 2.250000","moving_average","rxBytes"
588.700000,1112,168.200000,2018-04-04T10:47:26.000,7,"The device 1112 exceeded the grow up limit with value 3.500000","moving_variance","rxBytes"
58.000000,1112,23.200000,2018-04-04T10:47:31.000,7,"The device 1112 exceeded the grow up limit with value 2.500000","moving_average","rxBytes"
420.500000,1112,168.200000,2018-04-04T10:47:31.000,7,"The device 1112 exceeded the grow up limit with value 2.500000","moving_variance","rxBytes"
63.800000,1112,23.200000,2018-04-04T10:47:36.000,7,"The device 1112 exceeded the grow up limit with value 2.750000","moving_average","rxBytes"
69.600000,1112,23.200000,2018-04-04T10:47:41.000,7,"The device 1112 exceeded the grow up limit with value 3.000000","moving_average","rxBytes"
69.600000,1112,23.200000,2018-04-04T10:47:46.000,7,"The device 1112 exceeded the grow up limit with value 3.000000","moving_average","rxBytes"
63.800000,1112,23.200000,2018-04-04T10:47:51.000,7,"The device 1112 exceeded the grow up limit with value 2.750000","moving_average","rxBytes"
63.800000,1112,23.200000,2018-04-04T10:47:56.000,7,"The device 1112 exceeded the grow up limit with value 2.750000","moving_average","rxBytes"
44.949999,1112,21.749999,2018-04-04T10:48:01.000,7,"The device 1112 exceeded the grow up limit with value 2.066667","average","rxBytes"
69.600000,1112,23.200000,2018-04-04T10:48:01.000,7,"The device 1112 exceeded the grow up limit with value 3.000000","moving_average","rxBytes"
